transport/grpc/stream.go

```diff
@@ -200,5 +201,13 @@ func toYARPCStreamError(err error) error {
 	if !ok {
 		code = yarpcerrors.CodeUnknown
 	}
-	return yarpcerrors.Newf(code, status.Message())
+	yarpcerr := yarpcerrors.Newf(code, status.Message())
+	if len(status.Details()) > 0 {
+		details, err := proto.Marshal(status.Proto())
```

Comment:
you should use the method `marshalError` from the same pkg here.

The code can be very similar as the one from the method `invokeErrorToYARPCError`

Reply:
Didn't use `marshalError` as the status type in this function is from grpc/status and `marshalError` expects gogo/status. Have changed the types and reused them now